include/wpe/view-backend.h

```diff
@@ -97,6 +99,53 @@ WPE_EXPORT
 void
 wpe_view_backend_set_input_client(struct wpe_view_backend*, const struct wpe_view_backend_input_client*, void*);
 
+/**
+ * wpe_view_backend_set_dom_fullscreen_client:
+ * @view_backend: (transfer none): The view backend to obtains events from.
+ * @client: (transfer none): Client with callbacks for the events.
+ * @userdata: (transfer none): User data passed to client callbacks.
+ * 
+ * Configure a @client with callbacks invoked for DOM fullscreen requests.
+ * 
+ * This function must be only used once for a given @view_backend, the client
+ * cannot be changed once it has been set.
+ *
+ * Since: 1.12
+ */
+WPE_EXPORT
+void
+wpe_view_backend_set_dom_fullscreen_client(struct wpe_view_backend*, const struct wpe_view_backend_fullscreen_client*, void*);
+
+/**
+ * wpe_view_backend_fullscreen_handler:
+ * @userdata: (transfer none): User data passed to the embedder.
+ * @enable: (transfer none): User data passed to the embedder.
+ * 
+ * Type of functions used by an embedder to implement fullscreening web views.
+ *
+ * Returns: a boolean indicating whether the operation was completed.
+ * 
+ * Since: 1.12
+ */
+typedef bool (*wpe_view_backend_fullscreen_handler)(void *userdata, bool enable);
+
+/**
+ * wpe_view_backend_set_platform_fullscreen_handler:
+ * @view_backend: (transfer none): The view backend to obtains events from.
+ * @handler: (transfer none): Function used by an embedder to implement fullscreening web views.
+ * @userdata: (transfer none): User data passed to the handler function.
+ * 
+ * Handler function set by an embedder to implement fullscreening web views.
+ * 
+ * This function must be only used once for a given @view_backend, the handler
+ * cannot be changed once it has been set.
+ *
+ * Since: 1.12
+ */
+WPE_EXPORT
+void
+wpe_view_backend_set_platform_fullscreen_handler(struct wpe_view_backend*, wpe_view_backend_fullscreen_handler handler, void* userdata);
```

Comment:
Same for platform here, just set_fullscreen_handler

Reply:
Done